docs/verify/building_noise_models.ipynb

```diff
@@ -526,7 +527,7 @@
       "source": [
         "### Note on non-local qubit quantum error\n",
         "\n",
-        "`NoiseModel` does not support addition of non-local qubit quantum errors. They should be handled outside of `NoiseModel`. That suggests you should write your own transpiler pass (`TransformationPass`) and run the pass just before running the simulator if you need to insert your quantum errors into your circuit under your own conditions. See `noise.passes.LocalNoisePass` for example of how to implement such a transpiler pass."
+        "`NoiseModel` does not support addition of non-local qubit quantum errors. They should be handled outside of `NoiseModel`. That suggests you should [write your own transpiler pass](/transpile/custom-transpiler-pass) (`TransformationPass`) and run the pass just before running the simulator if you need to insert your quantum errors into your circuit under your own conditions."
```

Comment:
`noise.passes.LocalNoisePass` has moved to `noise.LocalNoisePass`, which doesn't seem to include instructions any more.